kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
So, there is a comment here to tell people to turn it on for debugging, but the actual line was removed: https://github.com/ManageIQ/manageiq-appliance-build/blob/d0f4d921daf92d60c48a08bd9ea722a9942f76a2/kickstarts/base.ks.erb#L2

There had to be a reason for removing it.  If not, I'm fine with this change, but I would like to make sure before adding it back in.